app/jobs/email_petition_signatories.rb

```diff
@@ -0,0 +1,85 @@
+class EmailPetitionSignatories
+  class PleaseRetry < StandardError; end
+
+  def self.run_later_tonight(job_class, petition, requested_at, *extra_args)
```

Comment:
Yeah, you're right, those methods don't really belong here - having to supply the job class should've been a hint to me!  A previous version that I moved away from actually had ``EmailPetitionSignatories::Worker`` be the ``ActiveJob`` instance and the two current ``ActiveJob`` instances (``EmailDebateOutcomesJob`` and ``EmailThresholdResponseJob``) weren't actually ``ActiveJob`` subclasses at all, they just configured the ``EmailPetitionSignatories::Worker`` by passing in loads of arguments.  I didn't really like that (especially that something called ``BlahJob`` wasn't actually a Job) so moved away, but didn't refactor these two methods.  Will do!